src/components/Dashboard/TodoList.vue

```diff
@@ -609,19 +609,22 @@ export default class TodoList extends Vue {
     return sessionStorage.getItem('TEMP_REG_NUMBER')
   }
 
-  /** Whether filing an Alteration is required (ie, there is a Todo filing). */
-  get requiresAlteration (): boolean {
-    if (this.isBcCompany || this.isUlc) {
-      return this.getTasks.some(task => task.task?.filing?.header?.name === FilingTypes.ALTERATION)
```

Comment:
The original intent here was to return True for BC->BEN or ULC->BEN alterations.

Now this code fails for regular BC/ULC alterations... now fixed.

Reply:
Also I moved this logic to the item itself (in loadAlteration()).